public/pages/Indices/containers/Indices/Indices.tsx

```diff
@@ -261,7 +303,11 @@ export default class Indices extends Component<IndicesProps, IndicesState> {
                 buttonProps: {
                   fill: true,
                   onClick: () => {
-                    this.props.history.push(ROUTES.CREATE_INDEX);
+                    this.props.multiDataSourceEnabled
```

Comment:
Let's extract such logic to a util function?

Reply:
I realised that we may not need to even have this, because CreateIndex can update its own URL by getting data source props from the context.